backup/pvc/bin/backup.sh

```diff
@@ -4,14 +4,16 @@ set -eo pipefail
 
 [[ ! $# -eq 1 ]] && echo "Usage: $0 backup_number" && exit 1;
 [[ -z "${BACKUP_DIR}" ]] && echo "Required 'BACKUP_DIR' env not set" && exit 1;
+[[ -z "${BACKUP_TMP_DIR}" ]] && echo "Required 'BACKUP_TMP_DIR' env not set" && exit 1;
```

Comment:
`BACKUP_TMP_DIR=${BACKUP_TMP_DIR:-/tmp}`

Default value is better than forcing users to set additional env in Jenkins CR.